submissions/adeyemiJuwon/css/style.css

```diff
@@ -0,0 +1,280 @@
+/**********************************
+GENERAL
+***********************************/
+*{
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+
+}
+/**********************************
+  HEADING
+  ***********************************/
```

Comment:
I think you should have a body that that has the attributes that is used in all of the page like `line-height` that is used in all the selector.

Reply:
done